packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.scss

```diff
@@ -49,6 +68,11 @@
     px-1
     outline-none
     transition-default;
+  background-color: var(
```

Comment:
can we replace this with an assignment to `--calcite-button-background-color`?

The same goes for the other calcite-button tokens.